src/module.c

```diff
@@ -8982,6 +8982,8 @@ void VM_RegisterClusterMessageReceiver(ValkeyModuleCtx *ctx,
 /* Send a message to all the nodes in the cluster if `target` is NULL, otherwise
  * at the specified target, which is a VALKEYMODULE_NODE_ID_LEN bytes node ID, as
  * returned by the receiver callback or by the nodes iteration functions.
+ * This API  uses lesser overhead if the target server support the light message header type (~30B)
+ * or else it uses the regular message overhead (~2KB).
```

Comment:
After a completed rolling upgrade, all nodes use the same message type. I think this scenario with mixed nodes is details that we don't need to mention, as long as the users understand that it works.

Maybe we can mention something in the style we sometimes have in the docs, like "In Valkey 8.1 and later, bla bla bla"...?

```suggestion
 *
 * In Valkey 8.1 and later, the cluster protocol overhead for this message is
 * ~30B, to compare with earlier versions where it's ~2KB.
```